src/components/connection-modal/connection-modal.css

```diff
@@ -364,20 +364,25 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding-bottom: 12px;
+    padding-top: 1rem;
```

Comment:
moved the margin-top that various bottom-area items had (instructions, dots) to here.

Kept the padding-bottom.

Added padding-left and -right, just in case anything ever gets too close to the edge.